### PR TITLE
Add Access Graph team and recent projects

### PR DESCRIPTION
**Pull request by jakule**

### CODE REVIEW

**Review by ibeckermayer — APPROVED** [review]

File: teams.md [hunk 1]
```diff
@@ -125,6 +125,17 @@ Recent projects:
 * Allow users to customize terminal fonts and keyboard shortcuts
 * Add a new cross-cluster search experience
 
+### Access Graph
+
+The Access Graph team specializes in analyzing resource access patterns, 
+providing valuable insights and enhanced visibility into infrastructure 
+by examining various access paths.
```
Comment (r0mant) [inline teams.md:132]: I would rephrase a bit to clarify the team actually writes software for this analysis, and remove fluff words.

```suggestion
The team is responsible for the design and implementation of Access Graph,
a tool that provides Teleport users with insights and enhanced visibility
into their infrastructure by visualizing access paths to cluster resources
in a unified graph view with querying capabilities.
```

File: teams.md [hunk 2]
```diff
@@ -125,6 +125,17 @@ Recent projects:
...
+
+Recent projects:
+
+* Import of Teleport RBAC and AWS IAM into a unified representation.
+* Implementing SQL-like query language for querying the access paths.
```
Comment (r0mant) [inline teams.md:137]: I would mention something about UI here as well as this is where most innovation is happening it feels like, something like "Designing scalable graph UI that can display hundreds of resources while maintaining responsiveness and user-friendliness."

File: teams.md [hunk 3]
```diff
@@ -125,6 +125,17 @@ Recent projects:
 * Allow users to customize terminal fonts and keyboard shortcuts
 * Add a new cross-cluster search experience
 
+### Access Graph
```
[the remark this answers is not in the document]
Comment (jakule) [reply teams.md:128]: Good suggestion. We used to have the access graph-specific website, but it's now a part of Teleport policy. I added a link to it as I still think that is very relevant.

**Review by r0mant — APPROVED** [review]
